account_payment_return/models/__init__.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
```

Comment:
remove coding in all files (python 3)
Remove copyright info in all init.py files